tests/Zend/Filter/LocalizedToNormalizedTest.php

```diff
@@ -90,7 +90,7 @@ public function testDateNormalizationWithoutParameters()
                 'date_format' => 'dd.MM.y',
                 'locale'      => 'de',
                 'day'         => '20',
-                'month'       => '04',
+                'month'       => '4',
```

Comment:
I noticed this in `zf1-db` as well. PHPUnit 7.0 (lowest) and 7.5 (default) differ in how strict they are for numbers like this.

This is actually what this particular call to the filter is outputting, but in PHPUnit < 7.5 (not sure exactly when the changed happened) it doesn't fail ("04" and "4" are considered "equal"), but in 7.5 they're not.